slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -237,6 +237,45 @@ void SLAMServiceImpl::ProcessDataOffline() {
     CreateMap();
 }
 
+std::string SLAMServiceImpl::GetNextDataFile() {
+    if (offlineFlag) {
```

Comment:
[nit] Can you put the code in this code block within `if (offlineFlag)` into a function? Would make it easier for my brain to process it with two sub-functions.